torch_xla/core/xla_model.py

```diff
@@ -77,6 +77,8 @@ def is_xla_tensor(tensor):
 
 
 def parse_xla_device(device):
+  #TODO(zpcore): remove this function for release 2.5.
+  logging.warning("function parse_xla_device will be deprecated in release 2.5")
   m = re.match(r'([A-Z]+):(\d+)$', device)
```

Comment:
in here you can just call the internal version.

Reply:
Oh, nice catch!